includes/rules/possible_heading.php

```diff
@@ -111,14 +111,14 @@ function edac_css_font_size_weight_check( $content ) {
 						if ( array_key_exists( 'font-weight', $rules ) ) {
 							// replace CSS variables.
 							$rules['font-weight'] = edac_replace_css_variables( $rules['font-weight'], $css_array );
-							if ( in_array( $rules['font-weight'], array( 'bold', 'bolder', '700', '800', '900' ) ) ) {
+							if ( in_array( $rules['font-weight'], array( 'bold', 'bolder', '700', '800', '900' ), true ) ) {
 								$has_bold_or_italic = true;
 							}
 						}
 						if ( array_key_exists( 'font-style', $rules ) ) {
 							// replace CSS variables.
 							$rules['font-style'] = edac_replace_css_variables( $rules['font-style'], $css_array );
-							if ( in_array( $rules['font-style'], array( 'italic', 'oblique' ) ) ) {
+							if ( 'italic' === $rules['font-style'] || 'oblique' === $rules['font-style'] ) {
```

Comment:
`in_array()` is a relatively slow function - like all the array operations in PHP, and should be avoided if possible. This was a simple change since we just want to check if the font-style is one of 2 values.